units/src/amount/signed.rs

```diff
@@ -50,17 +50,17 @@ pub struct SignedAmount(i64);
 
 impl SignedAmount {
     /// The zero amount.
-    pub const ZERO: Self = SignedAmount(0);
+    pub const ZERO: Self = SignedAmount::from_sat_unchecked(0);
```

Comment:
When `unwrap()` is available, I think we should just use that.  Although it doesn't look like that will be until MSRV 1.83 https://doc.rust-lang.org/stable/core/option/enum.Option.html#method.unwrap

Reply:
In the past we have just added TODOs in the code, although preferably there's a better way to remind future us to do change these.